hri_safety_sense/src/JoystickHandler.h

```diff
@@ -18,34 +18,40 @@
 /**
```

Comment:
I'd suggest renaming this file to `JoystickHandler.hpp`, which will allow us to use the linters more correctly.

Reply:
7de7162